SUMMARY.md

```diff
@@ -310,7 +313,6 @@
 ## WebRTC Calling and Video API
 * [Overview](webrtc/about.md)
 * [WebRTC Concept Overview](webrtc/guides/overview.md)
-* [Quickstart Guide](webrtc/guides/quickstart.md)
```

Comment:
Can you revert this? I don't know why it fails locally but it's fine to keep it on the live site

Reply:
good catch